packages/mobile/App/models/PatientProgramRegistrationCondition.ts

```diff
@@ -41,11 +41,7 @@ export class PatientProgramRegistrationCondition
   @RelationId(({ patientProgramRegistration }) => patientProgramRegistration)
   patientProgramRegistrationId: ID;
 
-  @ManyToOne(
-    () => ProgramRegistryCondition,
-    ({ patientProgramRegistrationConditions }) => patientProgramRegistrationConditions,
-    { nullable: true },
-  )
+  @ManyToOne(() => ProgramRegistryCondition, ({ conditions }) => conditions, { nullable: true })
```

Comment:
un-related fix to model alias. We are not actually using this yet but still it is incorrect.